query-node/mappings/src/workingGroup.ts

```diff
@@ -0,0 +1,165 @@
+import { SubstrateEvent } from '@dzlzv/hydra-common'
+import { DatabaseManager } from '@dzlzv/hydra-db-utils'
+import { FindConditions } from 'typeorm'
+import { Bytes } from '@polkadot/types'
+
+import {
+  inconsistentState,
+  logger,
+} from './common'
+
+import {
+  Channel,
+  Worker,
+  WorkerType,
+} from 'query-node'
+import {
+  GatewayWorkingGroup,
+  StorageWorkingGroup,
+} from '../../generated/types'
+import {
+  ApplicationId,
+  ApplicationIdToWorkerIdMap,
+  WorkerId,
+} from "@joystream/types/augment";
+
+/////////////////// Storage working group //////////////////////////////////////
+
+export async function storageWorkingGroup_OpeningFilled(db: DatabaseManager, event: SubstrateEvent): Promise<void> {
+  // read event data
+  const {applicationIdToWorkerIdMap} = new StorageWorkingGroup.OpeningFilledEvent(event).data
+
+  // call generic processing
+  await workingGroup_OpeningFilled(db, WorkerType.STORAGE, applicationIdToWorkerIdMap)
+}
+
+export async function storageWorkingGroup_WorkerStorageUpdated(db: DatabaseManager, event: SubstrateEvent): Promise<void> {
+  // read event data
+  const {workerId, bytes: newMetadata} = new StorageWorkingGroup.WorkerStorageUpdatedEvent(event).data
+
+  // call generic processing
+  await workingGroup_WorkerStorageUpdated(db, workerId, newMetadata)
```

Comment:
you are passing worker id as the unique id, you should be passing the worker type as well or derive the unique ID here for the worker type and pass it.

Reply:
Same problem in the following functions event handlers`_TerminatedWorker`, _WorkerExited, `_WorkerStorageUpdated`

Reply:
You're right, part of implementation was missing there. Fixed